howto/troubleshoot/bug-report.md

```diff
@@ -0,0 +1,21 @@
+*Applies to: Anbox Cloud, Anbox Cloud Appliance since 1.16.0*
+
+In some cases, you might encounter issues with Anbox Cloud that you are not able
+to troubleshoot yourself, or that might be bugs in the software itself. In this
+case, get in touch with the Anbox Cloud support team. They will ask you to
+create a bug report for your issue.
```

Comment:
The support team wont ask for a bug report but a support ticket. 

A customer may not have access to support (e.g. if they bought the appliance through the AWS marketplace which comes without any support attached).

Reply:
So, if a customer does not have access to support, would they need to run the tool at all?

Reply:
Would it work for you if we changed the last two sentences to something like:
"In this case, the Anbox Support team might ask you to generate a container bug report."

If the customer has access to support, they _might_ (or may?) ask for it, but they also might not, depending on the case.

Reply:
If a customer has no access to support they can still go and file a bug report at https://bugs.launchpad.net/anbox-cloud where we would work on a best-effort basis and ask maybe to run `anbox-bug-report` depending on the bug. The same applies to a support case. Support will evaluate the problem and then maybe ask the customer to run `anbox-bug-report`.

Reply:
> In this case, the Anbox support team or the engineering team might ask you to generate a bug report.

Given that you'd like this document to contain more than just container bug reports, keeping it a bit vague a that point seems preferable.


We could be more explicit about not running it if not needed, with something like:
> Please only follow these steps when asked by the Anbox support team or the engineering team.

Or is that too direct?

Reply:
I would not give directions what to do and what not to do but rather point out the possible way to collect logs / information for different parts of the stack. That way whoever needs to tell someone can easily link to the right section.

Reply:
I agree - right now we're using "bug report" both for the thing that customers file and for what they might be asked to attach. That is confusing.

Reply:
Resolved for now as we agreed to think about this after the release.